velox/expression/CastExpr.cpp

```diff
@@ -452,6 +452,62 @@ VectorPtr CastExpr::applyRow(
       std::move(newChildren));
 }
 
+template <typename FROM, typename T>
```

Comment:
Probably use `TOutput` `TInput` as the template names to match other usages.

Reply:
Done.